src/index.ts

```diff
@@ -336,6 +337,44 @@ function suppressNoSubscriptionExistsWarning<T>(f: () => T): T {
   }
 }
 
+/**
+ * Another VERY hacky monkeypatch to make sure that we can take extra parameters to certain alchemy subscriptions
+ * I hate doing this, but the other option is to fork web3-core and I think for now this is better. Any web3-core
+ * update to this method will have to be manually duplicated by us here.
+ */
+const web3CoreSubscriptions = require("web3-core-subscriptions"); // tslint:disable-line:no-var-requires
+const web3CoreErrors = require("web3-core-helpers").errors; // tslint:disable-line:no-var-requires
```

Comment:
TSLint wants you to write this with import statements and put them with the others at the top of the file
```ts
import web3CoreSubscriptions from "web3-core-subscriptions";
import { errors as web3CoreErrors } from "web3-core-helpers";
```
Any particular reason not to put them there? If so, would be good to add a comment for why we're using a different import style here.

Reply:
I was not able to figure out why, but the subscription prototype was not accessible when importing. Would be great if you can figure out how to make that work.

Reply:
Added a note and merging for now in order to resolve the end-user problem. We can revisit if desired.